src/web.ts

```diff
@@ -67,11 +69,13 @@ export class SentryCapacitorWeb
   /**
    *
    */
-  getStringBytesLength(payloadString: 'somepayload'): number {
+  getStringBytesLength(payload: {
+    string: string;
+  }): Promise<{ value: number }> {
     // TODO integrate web
     /* eslint-disable-next-line no-console */
-    console.log(payloadString);
-    return 12;
+    console.log(payload.string);
```

Comment:
Same here. 💣 this?

Reply:
@josh I was just using these temporarily to please the linter, but I can add an eslint exception instead if you'd prefer!